packages/context/src/context-listener.ts

```diff
@@ -10,7 +10,7 @@ import {ValueOrPromise} from './value-promise';
 /**
  * Context event types
  */
-export type ContextEventType = 'bind' | 'unbind';
+export type ContextEventType = string;
```

Comment:
Can we add some sort of documentation to describe well-known types? For example, list them in the tsdoc comment.

Personally, I was hoping that `'bind' | 'unbind' | string` will allow IDEs like VSCode to offer at least some sort of autocompletion to developers implementing listeners.

With a `string` type only, it's difficult to tell what are the valid event types.